src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
```

Comment:
The wrapper should have a maxium limit to which it can expand, otherwise it can't be used in any places that we need it to:
- action page feed
- details sidebar

This is how it looks when using it in the feed, with padding:

![Screenshot from 2021-03-12 12-32-38](https://user-images.githubusercontent.com/1193222/110928175-40469080-832f-11eb-8934-4b2ed8f3aa86.png)

Notice how it:
- breaks the container
- doesn't have right padding applied (because it doesn't have a limit)